.github/workflows/generate-osqueryd-targets.yml

```diff
@@ -44,7 +44,7 @@ jobs:
 
       - name: Download and extract osqueryd for linux
         run: |
-          curl -L https://pkg.osquery.io/linux/osquery-${{ env.OSQUERY_VERSION }}_1.linux_x86_64.tar.gz --output osquery-${{ env.OSQUERY_VERSION }}_1.linux_x86_64.tar.gz
+          curl -L https://github.com/osquery/osquery/releases/download/${{ env.OSQUERY_VERSION }}/osquery-${{ env.OSQUERY_VERSION }}_1.linux_x86_64.tar.gz --output osquery-${{ env.OSQUERY_VERSION }}_1.linux_x86_64.tar.gz
```

Comment:
Because faster to download github from github? :)

Reply:
Actually just because the packages get pushed to GitHub before they get pushed to pkg.osquery.io. Since 5.3.0 is in prerelease and I wanted to push it to the `edge` channel I needed to change the URLs.